### PR TITLE
VEN-605 | Update WSPlace generate script

### PR DESCRIPTION
**Pull request by a-rmz**
## Description :sparkles:
Update the script that imports `WinterStoragePlace`s based on the spreadsheet on the ticket's attachments

### Closes :no_good_woman:
**[VEN-605](https://helsinkisolutionoffice.atlassian.net/browse/VEN-605):** update WS places generating script

### Automated tests :gear:️
```shell
$ python resources/tests/test_resources_commands.py
```

### Manual testing :construction_worker_man:
1. Run the command
```
$ python manage.py generate_winter_places
```
2. Check the [`resources/winterstorageplace`](http://localhost:8000/admin/resources/winterstorageplace/)

### CODE REVIEW

File: resources/management/commands/generate_winter_places.py [hunk 1]
```diff
@@ -14,27 +14,42 @@
...
     "Iso-Sarvasto": [
-        ((1, 29), 2.5, 6.0),  # ((from_place_number, to_place_number), width, length)
-        ((30, 69), 3.0, 8.0),
-        ((70, 85), 3.5, 10.0),
-        ((86, 89), 4.5, 12.0),
+        ("A", (1, 14), 2.5, 6.0,),
+        ("B", (15, 22), 3.0, 8.0),
+        ("B", (23, 23), 2.5, 6.0,),
+        ("C", (24, 37), 2.5, 6.0,),
+        ("ALAKENTTÄ", (1, 19), 3.0, 8.0),
```
Comment (igordavydsson) [inline resources/management/commands/generate_winter_places.py:24]: nitpick, but I would keep the identifier in pascal case, i.e. `Alakenttä`, we already have them like that for piers (e.g. `Rantamuuri`)

File: resources/tests/test_resources_commands.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import pytest  # noqa
+from django.core.management import call_command
+from munigeo.models import Municipality
+
+from ..management.commands.generate_winter_places import WINTER_AREAS_MAP
+from ..models import WinterStorageArea, WinterStoragePlace, WinterStorageSection
+
+
+def test_generate_winter_places():
```
Comment (igordavydsson) [inline resources/tests/test_resources_commands.py:9]: wow, neat test 😎 👍

File: resources/management/commands/generate_winter_places.py (hunk 1 above)
Comment (a-rmz) [reply resources/management/commands/generate_winter_places.py:24]: Right! Didn't notice that 👍

**Review by igordavydsson — APPROVED** [review]
good stuff

**Pull request merged**